doc/api/process.md

```diff
@@ -129,7 +129,8 @@ unforeseen and unpredictable issues.
 
 Exceptions thrown from within the event handler will not be caught. Instead the
 process will exit with a non zero exit code and the stack trace will be printed.
-This is to avoid infinite recursion.
+This is to avoid infinite recursion. There may be other situations where the
```

Comment:
Maybe you should just amend the first sentence in the section that describes what `'uncaughtException'` event is. Currently it says:

> The 'uncaughtException' event is emitted when an exception bubbles all the way back to the event loop.

You could change it to say something like:

> The 'uncaughtException' event is emitted when a thrown JavaScript exception bubbles all the way back to the event loop.